webapp/models/secretStorage.js

```diff
@@ -0,0 +1,31 @@
+export default {
```

Comment:
From my POV it's better to separate constants based on domain like 
```
export const fbCredentials = {
  APP_ID: '266665100883899',
  APP_SECRET: '770ba6caab9a46e8f0476bcef87472cb',
}
```
and so on